google-http-client/src/main/java/com/google/api/client/http/ScopeFactory.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.api.client.http;
+
+class ScopeFactory {
+  /** This is used to check if the optional OpenCensus deps are on the classpath */
+  private static final boolean HAS_OPEN_CENSUS_CLASSES_ON_CLASSPATH = hasClasses(
+          "io.opencensus.trace.Span",
+          "io.opencensus.implcore.trace.RecordEventsSpanImpl",
+          "io.opencensus.contrib.http.util.HttpPropagationUtil"
+  );
+
+  private static NoopScope noopScope = new NoopScope();
+
+  static Scope getScope(Span span) {
+    if (HAS_OPEN_CENSUS_CLASSES_ON_CLASSPATH) {
+      return new OpenCensusScope((OpenCensusSpan) span);
+    } else {
+      return noopScope;
+    }
+  }
+
+  private static boolean hasClasses(String ...classNames) {
```

Comment:
not sure; is this usually String... classNames?

Reply:
Would you prefer `List`?

Reply:
The question was whether the space goes before or after the three dots by convention

Reply:
Awe I see... you are right. I ran formatted on all the new files. I filed an issue in this repo yesterday to get that to be a part of the build. A lot of files need to be formatted. Good catch!